README.md

```diff
@@ -0,0 +1,65 @@
+# Reverse chain adapter
+## Service overview 
+Reverse chain adapter is a service that is used for handling crucial financial operations in Iroha Hyperledger. 
+It uses RabbitMQ as middleware between Iroha clients and Iroha peer. 
+Using the service, transactions are not sent directly to Iroha peer but stored in a dedicated persistent queue. 
+Once a transaction is saved in the queue, the service will read the incoming transaction and try to send it to Iroha peer. 
+The service will send the transaction back to the queue if Iroha peer is not online, forming a loop. 
+The service will try to send every incoming transaction to Iroha peer unless it's not received no matter how many `send` attempts it might take. 
+The service will read all the incoming transactions from the queue on start-up and try to handle them if the service was offline by the time transaction was sent to RabbitMQ.
```

Comment:
all incoming transactions; if the service was offline when transaction was sent...